### PR TITLE
CORTX-31290: OVA Solution spec for v0.6.0

### PR DESCRIPTION
**Pull request by mukul-seagate11**
Signed-off-by: Mukul Malhotra <mukul.malhotra@seagate.com>

# Problem Statement
- OVA solution.yaml file for v0.6.0 where cortx-all image is removed

# Design
-  For Bug, Describe the fix here.
-  For Feature, Post the link for design

# Coding
   Checklist for Author
-  [x] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
- [ ] Unit and System Tests are added
- [ ] Test Cases cover Happy Path, Non-Happy Path and Scalability
- [ ] Testing was performed with RPM
- [x] Jenkins pipelines used for testing

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
- [ ] Interface change (if any) are documented
- [ ] Side effects on other features (deployment/upgrade)
- [ ] Dependencies on other component(s)

# Review Checklist 
  Checklist for Author
- [x] JIRA number/GitHub Issue added to PR
- [x] PR is self reviewed
- [x] Jira and state/status is updated and JIRA is updated with PR link
- [x] Check if the description is clear and explained

# Documentation
  Checklist for Author
- [ ] Changes done to WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

File: solutions/ova/kubernetes/ova-solution.yaml [hunk 1]
```diff
@@ -11,29 +11,26 @@ solution:
       csm_auth_admin_secret: seagate2
```
Comment (shailesh-vaidya) [inline solutions/ova/kubernetes/ova-solution.yaml:11]: Please remove all the secrets except below
```
      s3_auth_admin_secret: ldapadmin
      csm_mgmt_admin_secret: Cortxadmin@123
```

**Review by shailesh-vaidya — CHANGES_REQUESTED** [review]
Please fill PR template and add the Jenkins job used for validating this solution.yaml. 
Please add the Jira ticket in the PR title.

File: solutions/ova/kubernetes/ova-solution.yaml (hunk 1 above)
Comment (mukul-seagate11) [reply solutions/ova/kubernetes/ova-solution.yaml]: @shailesh-vaidya, solution.yaml is already validated for deployment i.e. https://eos-jenkins.colo.seagate.com/job/Cortx-Deployment/job/Nightly-K8s-Deploy/790/console

Comment (mukul-seagate11) [reply solutions/ova/kubernetes/ova-solution.yaml]: @shailesh-vaidya, This has been validated with jenkins job, https://eos-jenkins.colo.seagate.com/job/Cortx-Kubernetes/job/setup-cortx-cluster-solution-input/449/

**Review by shailesh-vaidya — APPROVED** [review]

**Review by gauravchaudhari02 — APPROVED** [review]

**Pull request merged**